psalm.xml

```diff
@@ -123,12 +123,20 @@
                 <!-- DBAL 3.2 forward compatibility -->
                 <referencedClass name="Doctrine\DBAL\Platforms\PostgreSQLPlatform"/>
                 <referencedClass name="Doctrine\DBAL\Platforms\SQLServerPlatform"/>
+
+                <!-- PHP 8.1 Enum support -->
+                <referencedClass name="BackedEnum"/>
+                <referencedClass name="ReflectionEnum"/>
```

Comment:
Is this still necessary now that we run Psalm with the PHP 8.1 language level?

Reply:
Nope, fixed, thanks!